hosts/build01/default.nix

```diff
@@ -1,12 +1,15 @@
 { inputs, ... }:
 {
   imports = [
-    inputs.srvos.nixosModules.hardware-hetzner-online-amd
-    inputs.self.nixosModules.disko-zfs
-    inputs.self.nixosModules.builder
     inputs.self.nixosModules.community-builder
+    inputs.self.nixosModules.disko-zfs
+    inputs.srvos.nixosModules.hardware-hetzner-online-amd
   ];
 
+  nix.settings.max-jobs = 24;
```

Comment:
Is this the same as number of cores for each machine?

Reply:
Yes.